compiler/tests-ocaml/lib-gc/test_gc.ml

```diff
@@ -0,0 +1,4 @@
+let () =
+  assert ((Gc.get ()).custom_minor_max_size = 0);
+  assert ((Gc.stat ()).forced_major_collections = 0);
+  assert ((Gc.quick_stat ()).forced_major_collections = 0)
```

Comment:
I would be nice to have a test that will fail next time one add another field on the ocaml side.

Reply:
this file is missing a header.